src/main/java/aspects/UserRestoreAspect.java

```diff
@@ -96,16 +96,15 @@ public void closeRestoreFactory(JoinPoint joinPoint) throws Throwable {
         restoreFactory.getSQLiteDB().closeConnection();
     }
 
-    private HqAuth getAuthHeaders(String domain, String username, String sessionToken) {
-        HqAuth auth;
-        if (sessionToken != null && sessionToken.equals(authKey)) {
-            auth = new BasicAuth(touchformsUsername, touchformsPassword);
-        } else if (UserUtils.isAnonymous(domain, username)) {
+    private HqAuth getHqAuth(String domain, String username, String sessionToken) {
+        HqAuth auth = null;
+        if (UserUtils.isAnonymous(domain, username)) {
```

Comment:
can this be removed now that [this](https://github.com/dimagi/commcare-hq/pull/21006) has been merged, or is this something different?

Reply:
@wpride has that in a separate PR: https://github.com/dimagi/formplayer/pull/665